frontend/src/app/app-routing.module.ts

```diff
@@ -15,17 +15,16 @@ const routes: Routes = [
     canActivate: [AuthGuardService],
   },
   {
-    path: 'claims',
-    component: ViewClaimsComponent,
+    path: 'claims/:vin',
```

Comment:
Angular was confusing the `claims/:vin` and `claims/new` routes so the reorder fixes that.